meta-resin-common/classes/image-resin.bbclass

```diff
@@ -45,6 +45,14 @@ init_config_json() {
    # Default no to persistent-logging
    echo $(cat ${1}/config.json | jq ".persistentLogging=false") > ${1}/config.json
 
+   # Find board json and extract slug
+   common_path=$(echo "${BBLAYERS}" | grep -o -E '[^ ]+meta-resin-common')
+   json_path=${common_path}/../../../${MACHINE}.json
+   slug=$(jq .slug $json_path)
+
+   # Set deviceType for supervisor
+   echo $(cat ${1}/config.json | jq ".deviceType=$slug") > ${1}/config.json
```

Comment:
Well, I tried it locally on my machine, not from within the build system, and it was throwing an error, that's why I asked